inc/template-tags.php

```diff
@@ -448,12 +448,12 @@ function twentytwenty_body_classes( $classes ) {
 		}
 
 		// Check whether the current page should have an overlay header.
-		if ( is_page_template( array( 'template-cover.php' ) ) ) {
+		if ( is_page_template( array( 'templates/template-cover.php' ) ) ) {
 			$classes[] = 'overlay-header';
 		}
 
 		// Check whether the current page has full-width content.
-		if ( is_page_template( array( 'template-full-width.php' ) ) ) {
+		if ( is_page_template( array( 'templates/template-full-width.php' ) ) ) {
```

Comment:
Can't we just use the existing classes added for page templates?

Reply:
Do you mean moving the files into the `classes` folder instead of into a `templates` folder?

Reply:
No. I mean that WP already puts the page template slug as a class on the body. Why can't we just use that class instead of adding more?

Reply:
And do they really need to be body classes?

Reply:
I believe @andersnoren added them initially so that folks can target these pages via CSS later on. But you're correct. When selecting the template `Full Width Template` the classes `page-template-template-full-width` and `page-template-template-full-width-php` are already added to the body class. I'll update the PR and remove them for now.